crawler/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
+target/
```

Comment:
I wonder if this file is necessary since you have a project wide `.gitignore` file

Reply:
I was wondering about that! I was thinking maybe it's good practice to have one in case people contribute to a single module. I'll delete it for now!